doc/project/checklist.md

```diff
@@ -184,6 +179,20 @@ Any other software-visible design changes have been reviewed by the software tea
 
 All known "Won't Fix" bugs and "Errata" have been reviewed by the software team.
 
+### SEC_CM_IMPLEMENTED
+
+Any appropriate security counter-measures are implemented.
+For redundantly encoded FSMs, [the sparse-fsm-encode.py script](https://github.com/lowRISC/opentitan/blob/master/util/design/sparse-fsm-encode.py) must be used to generate the encoding.
+
+### SEC_NON_RESET_FLOPS
+
+A review of sensitive security-critical storage flops was completed.
+Where appropriate, non-reset flops are used to store secure material.
```

Comment:
is there any block we have implemented this? Is OTP/flash memory considered as this kind of security-critical storage flops?

Reply:
@cdgori @tjaychen are the non-reset flops still something we need to have on this checklist? 
or are we just standardizing on using reset everywhere now (and punch the reset upon scan entry)?